test/browser/test_force_exit.c

```diff
@@ -10,31 +10,19 @@
 #include <assert.h>
 #include <emscripten.h>
 
-int result = 0;
-
-void EMSCRIPTEN_KEEPALIVE success() {
-  printf("success? %d\n", result);
-  assert(result == 10);
-}
-
 void EMSCRIPTEN_KEEPALIVE later() {
   printf("later, now force an exit\n");
-  result += 10;
-  emscripten_force_exit(result);
+  emscripten_force_exit(0);
 }
 
 int main() {
-  atexit(success);
```

Comment:
This seems to no longer test that the atexit is executed?

Reply:
I'm not sure the test was checking that before since the result would be reported to the server by the onExit handler which was triggered by `emscripten_force_exit`. 

I guess that atexit handler would be called before the onExit.. but I"m not sure this test verifies that.   I think maybe I can update it so that we check for the output messages.

Reply:
I split this change out into #23597